### PR TITLE
Leejo/lpw2024 blog post

### PR DESCRIPTION
**Pull request by leejo**
Announce the London Perl & Raku Workshop 2024, add Lee's author info.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ You can propose an article as a [GitHub issue](https://github.com/tpf/perldotcom
 
 	% make new
 
-You can then check your article by starting a local server (using [Hugo 0.59](https://gohugo.io/news/0.59.0-relnotes/) and no other version):
+You can then check your article by starting a local server using [Hugo](https://gohugo.io/). Note that v0.59 of Hugo was previously recommended, however the repo here has confirmed to be working with versions as recent as v0.126. To start the server run:
```
Comment (oalders) [inline CONTRIBUTING.md:14]: I think the issue may not be in the build, but rather in the render. That is, I don't think anyone has taken the time to prove that the render is correct on later versions. Having said that, we do use a pinned version to deploy and as long as the new articles don't do anything unorthodox, this should probably be fine.

File: content/article/announcing-the-london-perl-raku-workshop-2024-lpw-.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+* **When**: Saturday 26th October 2024
+* **Where**: The Trampery, 239 Old Street, London EC1V 9EY
+
+This year's workshop will be held at [The Trampery, Old Street](https://thetrampery.com/spaces/venue-hire/), a dedicated modern event space in central London. We have hired both The Ballroom and The Library; allowing us to run a main track for up to 160 attendees, and second smaller track for up to 35 attendees.
+
+The Trampery is located a two minute walk from the Northern Line's Old Street tube station in central London. The Northern Line has stops at most of the major train stations in London, or trivial links to others, so we recommend taking the tube to get to the venue.
+
+### Signup & Submit Talks
+
+If you haven't already, please signup and submit talks using the [official workshop site](https://act.yapc.eu/lpw2024/)
```
Comment (oalders) [inline content/article/announcing-the-london-perl-raku-workshop-2024-lpw-.md:26]: s/signup/sign up/

**Review by oalders — CHANGES_REQUESTED** [review]
Looks great. Just the one issue.

File: CONTRIBUTING.md (hunk 1 above)
Comment (leejo) [reply CONTRIBUTING.md:14]: I'll add this detail to the update 👍

**Pull request merged**